inspector-axon/src/main/java/io/axoniq/inspector/client/RSocketHandlerRegistrar.kt

```diff
@@ -32,12 +32,12 @@ class RSocketHandlerRegistrar(
     private val handlers: MutableList<RegisteredRsocketMessageHandler> = mutableListOf()
 
     fun registerHandlerWithoutPayload(route: String, handler: () -> Any) {
-        logger.info("Registered Inspector handler for route {}", route)
+        logger.debug("Registered Inspector handler for route {}", route)
```

Comment:
Similarly debatable whether this has anything to do with "Add call to retrieve Segment status from the TokenStore", although less of a predicament compared to `workspaceId` removal from the token, IMO.

Reply:
I removed the logging changes from this PR.